packages/ra-example/posts.js

```diff
@@ -153,27 +153,16 @@ export const PostCreate = ({ ...props }) => (
         <SimpleForm
             toolbar={<PostCreateToolbar />}
             defaultValue={{ average_note: 0 }}
-            validate={values => {
```

Comment:
not good, please use another view for debugging field-level validation, we need this one to test form-level validation